server/trunk.lua

```diff
@@ -1,4 +1,3 @@
-local QBCore = exports['qb-core']:GetCoreObject()
```

Comment:
careful, you are still using qb_core functions on line 15.

Reply:
It's defined as a global variable in server/main.lua tho

Reply:
Urite